### PR TITLE
Feature/data page

### PR DESCRIPTION
**Pull request by j8seangel**
image
Data page which include:
- Intro component
- Tabs component
- Selects component
- Navigate through the filters using the new react router feature (please take a look [here](https://github.com/Vizzuality/national-resource-governance-index/compare/develop...feature/data-page?expand=1#diff-83adefa09316e8982163b7828bb59e86R31), it is using [this]( https://github.com/ReactTraining/react-router/blob/master/docs/API.md#withroutercomponent-options) feature

### CODE REVIEW

File: app/components/common/Select.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+function Select(props) {
+  const classNames = ['c-select'];
+  if (props.color) classNames.push(`-${props.color}`);
+
+  return (
+    <div className={classNames.join(' ')}>
+      {props.label &&
+        <span className="label">{props.label}</span>
+      }
+      <ReactSelect {...props} />
```
Comment (geriux) [inline app/components/common/Select.jsx]: 👏

File: app/routes/Routes.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <Redirect from="*" to="/" />
+    </Route>
+  </Router>
+);
+
+Routes.propTypes = {
+  history: React.PropTypes.object
+};
+
+export default connect()(Routes);
```
Comment (geriux) [inline app/routes/Routes.js]: Looks soo much better having other things in separate files like the shouldUpdateScroll helper, tracking of events etc!

**Comment by geriux** [comment]
Do I need to do something else besides `npm install` ? Because I'm getting an error and I don't see the data section as the screenshot attached

**Comment by j8seangel** [comment]
We need to set the .env file but for now I've removed the api request for fake data.
Please try it again and let me know if you still get the error.

**Pull request merged**